examples/https/package.json

```diff
@@ -33,14 +33,14 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^1.0.0",
-    "@opentelemetry/exporter-jaeger": "1.10.0",
-    "@opentelemetry/exporter-zipkin": "1.10.0",
-    "@opentelemetry/instrumentation": "0.36.0",
-    "@opentelemetry/instrumentation-http": "0.36.0",
-    "@opentelemetry/resources": "1.10.0",
-    "@opentelemetry/sdk-trace-base": "1.10.0",
-    "@opentelemetry/sdk-trace-node": "1.10.0",
-    "@opentelemetry/semantic-conventions": "1.10.0"
+    "@opentelemetry/exporter-jaeger": "1.10.1",
```

Comment:
Manually changing the versions indicate that there is no release automation on backport releases.
So it seems we have to take some extra care during the next main release to avoid that it selects the same versions for GA and/or experimental packages.